modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java

```diff
@@ -269,6 +288,170 @@ public void testDeleteAccountEntryByPrimaryKey() throws Exception {
 		_assertDeleted(accountEntry.getAccountEntryId());
 	}
 
+	@Test
+	public void testSearchFilterByAccountUserIds() throws Exception {
+		_addAccountEntries();
+
+		User user1 = UserTestUtil.addUser();
+		User user2 = UserTestUtil.addUser();
```

Comment:
We should add the users to `_users` so they can be removed after test.

Reply:
oops, good catch. Thanks!